src/Solnet.Examples/TransactionBuilderExample.cs

```diff
@@ -198,41 +204,87 @@ public void Run()
             Console.WriteLine($"MinBalanceForRentExemption Account >> {minBalanceForExemptionAcc}");
 
             var mintAccount = wallet.GetAccount(21);
-            Console.WriteLine($"MintAccount: {mintAccount.PublicKey}");
+            Console.WriteLine($"MintAccount: {mintAccount.PublicKey.Key}");

```

Comment:
Override the ToString method to return the Key property. It will make these debug messages less verbose